id_fishing_points_transit/results/readjsons.py

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/python
+import urllib2
+import json
+
+task_ids = [4108,4109,4110,4111,4112,4113,4114,4115,4116,4117,4118,4119,4120,4121,4122,4123,4124,4125,4126,4127,4128,4129,4130,4131,4132,4133,4134,4135,4136,4137,4138,4139,4140,4141,4142,4143,4144,4145,4146,4147,4148,4149,4150,4151,4152,4153,4154,4155,4156,4157,4158,4159,4160,4161,4162,4163,4164,4165,4166,4167,4168,4169,4170,4171,4172,4173,4174,4175,4176,4177,4178,4179,4180,4181,4182,4183,4184,4185,4186,4187,4188,4189,4190,4191,4192,4193,4194,4195,4196,4197,4198,4199,4200,4201,4202,4203,4204,4205,4206,4207,4208,4209,4210,4211,4212,4213,4214,4215,4216,4217,4218,4219,4220,4221,4222,4223,4224,4225,4226,4227,4228,4229,4230,4231,4232,4233,4234,4235,4236,4237,4238,4239,4240,4241,4242,4243,4244,4245,4246,4247,4248,4249]
+
+
+link = 'http://crowd.globalfishingwatch.org/project/maptests2/409/results.json'
+
+
+#users
+users = ["" for i in range(21)]
+users[1]='davidkroodsma'
+users[3]='davidkroodsmathe2nd'
+users[4]='alexwilson'
+users[5]='chris'
+users[2]='bjornbergman'
+users[6]='enriquetuya'
+users[7]='kristinaboerder'
+users[8]='vaiduke2'
+users[9]='katepepler'
+users[10]='stephanielewis'
+users[11]='AlexCerra'
+users[12]='juliecharbonneau'
+users[13]='ninagalle'
+users[14]='sidneyblack-rotchin'
+users[15]='daivdtest'
+users[16]='cailinburmaster'
+users[17]='elizabethnagel'
+users[18]='isabelfleisher'
+users[19]='ciarawillis'
+users[20]='clairechristie'
+
+user_count = [0 for i in range (21)]
+
```

Comment:
Should probably be: `user_count = [0 for _ in users]`  so that it can't get out of sync.